phpstan.neon

```diff
@@ -0,0 +1,52 @@
+includes:
+	- vendor/nunomaduro/larastan/extension.neon
+	- vendor/phpstan/phpstan-deprecation-rules/rules.neon
+	- vendor/thecodingmachine/phpstan-safe-rule/phpstan-safe-rule.neon
+	- vendor/phpstan/phpstan-strict-rules/rules.neon
+
+services:
+	-
+		class: PHPStan\Rules\StrictCalls\StrictFunctionCallsRule
+		tags:
+			- phpstan.rules.rule
+
+	-
+		class: PHPStan\Rules\SwitchConditions\MatchingTypeInSwitchCaseConditionRule
+		tags:
+			- phpstan.rules.rule
+
+parameters:
+	level: 7
+	paths:
+		- app
+	excludePaths:
+		- app/Relations/*
```

Comment:
What's so special about these? :smiley:

Reply:
they are annoying. :'D

Reply:
That's no reason to exclude them. So this means I need to work on them, right?

Reply:
> That's no reason to exclude them. So this means I need to work on them, right?

maybe :laughing: